docs/docs/guides/managing-appeals.md

```diff
@@ -23,149 +28,76 @@ This is the main use case of Guardian, to manage access approval for user to a p
 - Expire: If the appeal specifies the expiration policy then it will automatically get expired when it is already passed the lifetime limit.
 - Recreate: Possible for appeals that are currently still active, rejected, or terminated. This action will create a new appeal based on the previous one. For the appeal coming from active status, there is a policy related to access extension.
 
-To create an appeal, you can use this endpoint:
-
-```text
-POST /appeals
-Content-Type: application/json
-Accept: application/json
-
-Request Body:
-{
-   "user": "user@email.com",
-   "resources": [
-       {
-           "id": 1,
-           "role": "viewer",
-           "options": {
-             "duration": "24h"
-           }
-       }
-   ]
-}
-
-Response:
-[
-  {
-    "id": 1,
-    "resource_id": 1,
-    "policy_id": "bigquery_dataset",
-    "policy_version": 1,
-    "status": "pending",
-    "user": "user@email.com",
-    "role": "viewer",
-    "options": {
-      "expiration_date": "2021-05-10T09:49:26.402189+07:00"
-    },
-    "labels": null,
-    "approvals": [
-      {
-        "id": 11,
-        "name": "check_if_dataset_is_pii",
-        "appeal_id": 11,
-        "status": "pending",
-        "actor": null,
-        "policy_id": "bigquery_dataset",
-        "policy_version": 1
-      },
-      {
-        "id": 12,
-        "name": "supervisor_approval",
-        "appeal_id": 11,
-        "status": "pending",
-        "actor": null,
-        "policy_id": "bigquery_dataset",
-        "policy_version": 1,
-        "approvers": [
-          "john.doe@email.com"
-        ]
-      },
-      {
-        "id": 13,
-        "name": "admin_approval",
-        "appeal_id": 11,
-        "status": "pending",
-        "actor": null,
-        "policy_id": "bigquery_dataset",
-        "policy_version": 1,
-        "approvers": [
-          "owner@email.com"
-        ]
-      }
-    ]
-  }
-]
+### 1. Create an Appeal using CLI
```

Comment:
@rahmatrhd we can use docusaurus tabs feature here to switch between API use or CLI use. https://docusaurus.io/docs/markdown-features/tabs